src/test/java/org/opentripplanner/updater/alerts/AlertsUpdateHandlerTest.java

```diff
@@ -0,0 +1,91 @@
+package org.opentripplanner.updater.alerts;
+
+import com.google.transit.realtime.GtfsRealtime;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Spy;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.opentripplanner.routing.alertpatch.AlertPatch;
+import org.opentripplanner.routing.services.AlertPatchService;
+
+import java.util.Collection;
+import java.util.Date;
+import java.util.HashSet;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AlertsUpdateHandlerTest {
+
+    private AlertsUpdateHandler handler;
+
+    @Spy
+    private FakeAlertPatchService service;
+
+    @Before
+    public void setUp() {
+        handler = new AlertsUpdateHandler();
+        handler.setFeedId("1");
+        handler.setAlertPatchService(service);
+    }
+
+    @Test
+    public void testAlertWithTimePeriod() {
+        GtfsRealtime.Alert alert = GtfsRealtime.Alert.newBuilder()
```

Comment:
Can the content be refactored as follows?
```java
@Test
public void testAlertWithTimePeriod() {
    testAlertParameters(10, 20);
    // testAlertParameters(10, null), etc;
}
```

Reply:
Refactored in 95788c8c4b0c18065885ca7bd9af7e6c729e1bbf